styles/app.scss

```diff
@@ -5,4 +5,141 @@ $fa-font-path: '//netdna.bootstrapcdn.com/font-awesome/4.3.0/fonts';
 @import 'font-awesome/scss/font-awesome';
 @import 'box-sizing';
 
-@import 'adonis';
+body {
+  margin: 0;
+}
+
+
+//nav bar section
+.nav-bar {
+  display: flex;
```

Comment:
Good work using flex on this navigation, but try using `float` and then `inline-block` as practice to get a navbar that doesn't use flexbox.